src/ansys/dpf/core/operators/result/result_provider.py

```diff
@@ -94,12 +99,13 @@ def __init__(
         bool_rotate_to_global=None,
         mesh=None,
         result_name=None,
+        result_scripting_name=None,
         config=None,
         server=None,
     ):
-        super().__init__(name="custom", config=config, server=server)
-        self._inputs = InputsCustom(self)
-        self._outputs = OutputsCustom(self)
+        super().__init__(name="result_provider", config=config, server=server)
```

Comment:
@PProfizi added some time ago the very useful concept of alias operator internal names. The thing is that for old servers + new dpfcore the internal operator will be `"custom"` but the the API points to `result_provider`, which was not recorded, hence it will raise. The mechanism to prevent this is [here](https://github.com/ansys/pydpf-core/pull/2168). I think that if you add the entry `"result_provider": "custom"` to the map in build.py we will be good to go.

Reply:
done